SpatialGDK/Source/SpatialGDKFunctionalTests/SpatialGDK/UnresolvedReferenceGymTest/UnresolvedReferenceGymTest.h

```diff
@@ -0,0 +1,19 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#pragma once
+
+#include "CoreMinimal.h"
+#include "SpatialFunctionalTest.h"
+#include "UnresolvedReferenceGymTestActor.h"
+#include "UnresolvedReferenceGymTest.generated.h"
+
+UCLASS()
+class SPATIALGDKFUNCTIONALTESTS_API AUnresolvedReferenceGymTest : public ASpatialFunctionalTest
+{
+	GENERATED_BODY()
+
+public:
+	AUnresolvedReferenceGymTest();
+
+	virtual void PrepareTest() override;
```

Comment:
minor: could be final.
```suggestion
	void PrepareTest() final;
```